package.json

```diff
@@ -60,8 +60,10 @@
   },
   "devDependencies": {
     "@types/debug": "^4.1.7",
+    "@types/express-session": "^1.17.4",
```

Comment:
I believe all of these should actually be `dependencies` not `devDependencies`.

Reply:
Hi, the express-session is only used in tests. I will close this PR and reopen it on master.